File: Python编程从入门到实践/chapter06/动手试一试/6.py
favorite_languages = {
    'jen': 'python',
    'sarah': 'c',
    'edward': 'rust',
    'phil': 'python',
}

visited_names = ['jen', 'mayfly', 'sarah']
for name in visited_names:
    if name in favorite_languages.keys():
        print(f"Hell {name},Thank you!")
    else:
        print("Please visit our team")

File: C/C语言程序设计现代方法/chapter06/练习题/6.c
/*
 * Name: 6.c 
 * Purpose: 将1.c改为for语句
 * Author: mayfly
 */
#include <stdio.h>

int main(void)
{
  for (int i = 1;i <= 128;i *= 2)
  {
    printf("%d", i);
  }
}